3.10/aql/functions-arangosearch.md

```diff
@@ -277,20 +279,74 @@ by the specified **analyzer**.
   defaults to `"identity"`
 - returns nothing: the function evaluates to a boolean, but this value cannot be
   returned. The function can only be called in a search expression. It throws
-  an error if used outside of a [SEARCH operation](operations-search.html).
+  an error if used outside of a [`SEARCH` operation](operations-search.html) or
+  a `FILTER` operation that uses an inverted index.
 
 ```aql
 FOR doc IN viewName
   SEARCH EXISTS(doc.text, "analyzer", "text_en")
   RETURN doc
 ```
 
+#### Testing for nested fields
```

Comment:
BTW function EXISTS from the very beginning do support "type" keyword
like `EXISTS(d.foo, "type")`. Is it intentional to not document this?